src/test/java/com/exactpro/junit5/WithBufNameFormatter.java

```diff
@@ -0,0 +1,26 @@
+package com.exactpro.junit5;
+
+import io.netty.buffer.ByteBuf;
+
+import java.util.Collection;
+
+class WithBufNameFormatter {
+
+    static String formatName(String methodName, Collection<ByteBuf> arguments) {
+        StringBuilder displayName = new StringBuilder(methodName);
+        for (ByteBuf buf : arguments) {
+            displayName.append(" [").append(formatBuffer(buf)).append("]");
+        }
+        return displayName.toString();
+    }
+
+    private static String formatBuffer(ByteBuf buf) {
+        StringBuilder bufContent = new StringBuilder();
+        int readerIndex = buf.readerIndex();
+        int readLimit = buf.writerIndex();
+        while (readerIndex < readLimit) {
+            bufContent.append(Integer.toHexString(buf.getByte(readerIndex++) & 0b11111111)).append(" ");
```

Comment:
I'd use 0xFF here. It's easier to grasp than binary.
You shouldn't append " " after last element